### PR TITLE
PHOENIX-7221 Manage requests-gssapi version for Phython 3.7 and lower

### PR DESCRIPTION
**Pull request by stoty**

### CODE REVIEW

File: python-phoenixdb/setup.py [hunk 1]
```diff
@@ -50,8 +53,9 @@ def readme():
     install_requires=[
         'protobuf>=3.0.0',
         'requests',
-        'requests-gssapi',
-        'gssapi<1.6.0;python_version<"3.6"',
+        'requests-gssapi; python_version>="3.8.0"',
+        'requests-gssapi<1.30; python_version<"3.8.0"',
```
Comment (richardantal) [inline python-phoenixdb/setup.py:57]: shouldn't it be requests-gssapi<1.3.0

Comment (stoty) [reply python-phoenixdb/setup.py]: Thank you!
Fixed.

**Review by richardantal — APPROVED** [review]
LGTM

**Comment by stoty** [comment]
merged manually

**Pull request closed without merging**